EKrut-Client/src/gui/ClientLoginController.java

```diff
@@ -68,14 +70,16 @@ void attemptLogin(ActionEvent event) throws IOException {
 			return;
 		}
 		
-		
 		if (username.equals(password)) { // TBD IMPLEMENT REAL USERNAME & PASSWORD CHECKING
 			incorrectUserPassLbl.setVisible(true);
 			return;
 		}
-		
 		// LOGIN SUCCESS!
-		
 	}
+	
+	// Press enter from anywhere in scene in order to logged in.
+	public void enterKeyForLogin() {
+		loginBtn.setDefaultButton(true);
```

Comment:
If you set the default button property of the button in the FXML, this is unnecessary since you never change the default to anything else.